lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -23,6 +23,7 @@ def initialize(input, suspended_action:)
         @tags_flag = action_input[:tags_flag]
         @passphrase = action_input['secrets']['key_passphrase']
         @execution_timeout_interval = action_input[:execution_timeout_interval]
+        @cleanup_working_dirs = input.fetch(:cleanup_working_dirs)
```

Comment:
Ondra's right, it's better to have a fallback to default value, just to be safe
```suggestion
        @cleanup_working_dirs = input.fetch(:cleanup_working_dirs, true)
```